PEM - 9.1 Release branch (#3606)
EnterpriseDB/docs: product_docs/docs/pem/9/troubleshooting.mdx

Then try installing the PEM server again.

### RHEL 7 ppc64le

After installing the PEM server on RHEL 7 ppc64le, you may see the following cipher error in the worker log file:

```text
WARNING: ConnectToPEM: unable to connect to PEM database: could not create SSL context: library has no ciphers
```

> To resolve this error, you must upgrade your PostgreSQL to version 12.x, where x is the latest available version. Then, restart the PEM agent.

Reviewer: ```suggestion
To resolve this error, if you intend to stay on version PostgreSQL version 12, upgrade to the latest available version. Then, restart the PEM agent.
```
